AndroidTest/app/src/main/AndroidManifest.xml

```diff
@@ -19,10 +19,10 @@
         <activity
             android:name=".TestInformation"
             android:label="@string/title_activity_test_information"
-            android:parentActivityName=".MyActivity" >
+            android:parentActivityName=".TestActivity" >
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
-                android:value="cloudant.com.androidtest.MyActivity" />
+                android:value=".TestActivity" />
```

Comment:
This is now missing the qualifier `cloudant.com.androidtest` - is this an issue?